[rospy.client][INFO] 2021-09-10 14:48:38,040: init_node, name[/server_endpoint], pid[28787]
[xmlrpc][INFO] 2021-09-10 14:48:38,040: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2021-09-10 14:48:38,040: ROS Slave URI: [http://wng:44737/]
[xmlrpc][INFO] 2021-09-10 14:48:38,040: Started XML-RPC server [http://wng:44737/]
[rospy.impl.masterslave][INFO] 2021-09-10 14:48:38,040: _ready: http://wng:44737/
[rospy.registration][INFO] 2021-09-10 14:48:38,041: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-09-10 14:48:38,041: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-09-10 14:48:38,141: registered with master
[rospy.rosout][INFO] 2021-09-10 14:48:38,142: initializing /rosout core topic
[rospy.rosout][INFO] 2021-09-10 14:48:38,145: connected to core topic /rosout
[rospy.simtime][INFO] 2021-09-10 14:48:38,148: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-09-10 14:48:38,163: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-09-10 14:48:38,442: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-09-10 14:48:47,352: Connection from 192.168.1.146
[rosout][INFO] 2021-09-10 14:48:47,357: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-09-10 14:48:47,374: topic[/execution_check] adding connection to [http://wng:44737/], count 0
[rospy.internal][INFO] 2021-09-10 14:48:47,376: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-09-10 15:25:09,415: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-09-10 15:25:09,426: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][ERROR] 2021-09-10 15:25:09,426: Unable to initiate TCP/IP socket to wng:36199 (http://wng:44737/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2021-09-10 15:25:09,430: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-09-10 15:25:09,430: topic[/execution_check] removing connection to http://wng:44737/
[rospy.impl.masterslave][INFO] 2021-09-10 15:25:09,431: signal-2
[rospy.core][INFO] 2021-09-10 15:25:09,562: signal_shutdown [atexit]
